cli/testdata/coder_create_--help.golden

```diff
@@ -7,6 +7,9 @@ Create a workspace
      [40m [0m[91;40m$ coder create <username>/<workspace_name>[0m[40m [0m
 
 [1mOptions[0m
+      --parameter string-array, $CODER_RICH_PARAMETER
+          Rich parameter value in the format "name=value".
+
       --rich-parameter-file string, $CODER_RICH_PARAMETER_FILE
```

Comment:
> It might be a bit confusing to have one rich and one regular parameter flag.

TBH there are no legacy parameters anymore, just rich ones.

> Should we rename/alias this flag (now or in future)?

In theory, we should simplify it, but maybe extract it to a separate PR, more transparent than the refactor. I guess that this advice can be applied to other places too: `--rich-parameter-file`, `$CODER_RICH_PARAMETER_FILE`, `$CODER_RICH_PARAMETER`.